### PR TITLE
Rewritten the PPTP server documentation

### PR DESCRIPTION
**Pull request by aapostoliuk**
Fully rewritten PPTP server documentation.

### CODE REVIEW

File: docs/configuration/vpn/pptp.rst [hunk 1]
```diff
@@ -1,52 +1,550 @@
...
+  set vpn pptp remote-access authentication radius server 10.0.0.2 key 'foo'
+
+.. note:: Some RADIUS severs use an access control list which allows or denies
+   queries, make sure to add your VyOS router to the allowed client list.
+
+RADIUS source address
+=====================
+
+If you are using OSPF as IGP, always the closest interface connected to the
+RADIUS server is used. With VyOS 1.2 you can bind all outgoing RADIUS requests
```
Comment (rebortg) [inline docs/configuration/vpn/pptp.rst:92]: VyOS 1.2 ?
either delete or "since VyOS 1.2"?

**Review by rebortg — CHANGES_REQUESTED** [review]

File: docs/configuration/vpn/pptp.rst [hunk 2]
```diff
@@ -1,52 +1,550 @@
...
+      starting: 0
+      active: 1
+
+***************
+Troubleshooting
+***************
+
+.. code-block:: none
+
+    Feb 29 14:58:57 vyos accel-pptp[4629]: pptp: new connection from 192.168.10.100
```
Comment (rebortg) [inline docs/configuration/vpn/pptp.rst:488]: a command is missing

**Review by rebortg — CHANGES_REQUESTED** [review]

File: docs/configuration/vpn/pptp.rst (hunk 2 above)
Comment (aapostoliuk) [reply docs/configuration/vpn/pptp.rst:490]: done

File: docs/configuration/vpn/pptp.rst (hunk 1 above)
Comment (aapostoliuk) [reply docs/configuration/vpn/pptp.rst]: done. I have deleted '1.2'

**Pull request merged**